servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -68,13 +68,11 @@ def update_job_labels(id, body, **kwargs):
 
     # Follow API spec
     result = response.json()
-    all_labels = get_job(id).labels
-    if not all_labels:
-        all_labels = {}
+    if not result.get('labels'):
+        result.get('labels').update({})
 
     # Redundantly update all_labels with updated labels to provide consistency guarantees
```

Comment:
Should we remove this comment, since the latest Cromwell returns all the labels for the `PATCH` now?